_pages/03_now.md

```diff
@@ -1,11 +1,21 @@
 ---
 layout: page
-title: Now
```

Comment:
Removing this removes "Now" from the menu.

Reply:
oh Yes I just remove it cause it it was Breadcrumbs navigation but I'll back it sorry

Reply:
👌